docs/folders.md

```diff
@@ -116,14 +116,14 @@ folder and all of its contents, set the optional `recursive` parameter to `true`
 await client.FoldersManager.DeleteAsync("11111", recursive: true);
 ```
 
-Create a Shared Link for a Folder
+Create or update a Shared Link
 ---------------------------------
 
-You can create a shared link for a folder by calling
+You can create or update a shared link for a folder by calling
 `FoldersManager.CreateSharedLinkAsync(string id, BoxSharedLinkRequest sharedLinkRequest, IEnumerable<string> fields = null)`
 with the ID of the folder and the shared link parameters.
 
-<!-- sample put_folders_id_shared_link_create -->
+<!-- sample put_folders_id shared_link_create -->
```

Comment:
The same question as previously, is `put_folders_id shared_link_create` should be here?
Or maybe we should use `'put_folders_id add_shared_link` instead?

Reply:
Fixed